asn1ate/sema.py

```diff
@@ -217,7 +216,9 @@ def children(self):
         sema nodes.
         """
         # Collect all SemaNode members.
-        members = list(vars(self).values())
+        # We need to exclude the 'referenced_modules' member if it exists
+        members = [v for k,v in vars(self).iteritems() if k != 'referenced_modules']
```

Comment:
Actually, now that I see this -- this should probably be handled below, for the list members, right?